src/ddmd/root/rmem.d

```diff
@@ -13,6 +13,8 @@ module ddmd.root.rmem;
 // Online documentation: https://dlang.org/phobos/ddmd_root_rmem.html
 
 import core.stdc.string;
+import core.stdc.stdlib;
+import core.stdc.stdio;
```

Comment:
While most files at DMD still use full "global" imports, there's no reason not to start using either local imports, selective ones or even both ;-)

Reply:
Ok, amended. Also added calling GC.free, no reason to omit it.